packages/aws_common/pubspec.yaml

```diff
@@ -12,6 +12,7 @@ dependencies:
   collection: ^1.15.0
   http: ^0.13.4
   meta: ^1.7.0
+  uuid: ^3.0.1
```

Comment:
I would think this would stay in amplify_core - what was the intention here?

Reply:
My thinking was that aws_common could have any generic utils that do not have dependencies on Flutter or other Amplify-related types.

Reply:
Gotcha- sounds good thanks. I'll want to change the interface and add doc comments but I'll do that in a follow up PR